src/test/java/webdrivertasks/waitings/WebElementWaiting.java

```diff
@@ -0,0 +1,35 @@
+package webdrivertasks.waitings;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class WebElementWaiting {
+
+    private static final long TIME_OUT = 20;
```

Comment:
1.since there is no way to change this value, we could use int type
2.I would also think about the name. Perhaps, smth like `DEFAULT_WAIT_TIMEOUT_SECONDS` would fit better